.travis.yml

```diff
@@ -15,6 +15,7 @@ install:
   - pip install -r requirements.txt
   - pip install sphinx
   - pip install sphinx_rtd_theme
+  - pip install docutils==0.12 
```

Comment:
Travis was failing because of this error: 
```
assert not self.context, 'len(context) = %s' % len(self.context)
AssertionError: len(context) = 1```

and i just followed the sphinx issues where they said you need to also install this.